src/api/physicalPersonService.ts

```diff
@@ -0,0 +1,30 @@
+import { PhysicalPersonType } from '../components/shared/types/physical-person'
+import client from './axios'
+
+const baseUrl = '/physical-person'
+
+const findById = async (id: number): Promise<PhysicalPersonType> => {
+  const result = await client.get(`${baseUrl}/${id}`)
+  return result.data
+}
+
+const findAll = async (): Promise<PhysicalPersonType[]> => {
+  const result = await client.get(baseUrl)
+  return result.data
+}
+
+const save = async (physicalPerson: PhysicalPersonType): Promise<PhysicalPersonType> => {
+  const result = await client.post(baseUrl, physicalPerson)
+  return result.data
+}
+
+const deleteOneById = async (id: number): Promise<void> => {
+  return await client.delete(`/physical-person/${id}`)
+}
```

Comment:
Aqui tu pode decidir o seguinte: quem trata a exceção? Se vc colocar um `await` nos requests, vc vai retornar uma promise da resposta. E o código cliente (o codigo que vai consumir essas funções) vai precisar tratar as exceptions. Ou vc trata as exceptions aqui ou no codigo cliente. Se o codigo cliente for tratar, pode retornar a promise do request ao inves de dar o await. Se vc tratar as exceptions no service, vc precisa implementar os codigos de tratamento de erro.